tests/CliTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Symfony\Component\Console\Tester\ApplicationTester;
+
+/**
+ * @requires PHP >= 8.0
+ */
+class CliTest extends Yoast\PHPUnitPolyfills\TestCases\TestCase
+{
+    public function prepTestConfig()
```

Comment:
This can move into `setUp` if we want, just wanted to separate out this functionality while I was proving it out.

Reply:
Yeah I think moving this into setUp is the right call.